src/orbit.rs

```diff
@@ -188,26 +269,28 @@ async fn load_orbit_(oid: Cid, dir: PathBuf) -> Result<SimpleOrbit> {
     })
 }
 
-pub fn verify_oid(oid: &Cid, pkh: &str, uri_str: &str) -> Result<()> {
-    // try to parse as a URI with matrix params
-    let mut parts = uri_str.split(';');
+pub fn get_oid_matrix_params<'a>(
```

Comment:
Should `OrbitId` be its own type instead so we're sure it is well formed (e.g. it has a method) on the verification/deserialisation step?